### PR TITLE
feat: google analytics for scene

### PR DESCRIPTION
**Pull request by HideBa**
* Added google analytics properties on Scene

### CODE REVIEW

File: pkg/builtin/manifest.yml [hunk 1]
```diff
@@ -139,6 +139,19 @@ extensions:
               description: "Set saturation of the fog. Min: -1 Max: 1"
               min: -1
               max: 1
+        - id: googleAnalytics
+          title: Google Analytics
+          description: Set your Google Analytics tracking ID and analize how your published project being viewed.
```
Comment (KaWaite) [inline pkg/builtin/manifest.yml:144]: Set your......and analyze how your published project is being viewed.

File: pkg/builtin/manifest.yml [hunk 2]
```diff
@@ -139,6 +139,19 @@ extensions:
...
+          fields:
+            - id: enableGA
+              type: bool
+              title: Enable
+              defaultValue: false
+              description: Enable Google Analytics
+            - id: trackingId
+              type: string
+              title: Tracking ID
+              description: Paste your Google Analytics tracking ID here. This would be mebeded in your published project.
```
Comment (KaWaite) [inline pkg/builtin/manifest.yml:154]: ..... . This will be embedded in your published project

**Review by KaWaite — COMMENTED** [review]
Where are we embedding it into the project? I see you are adding the options to choose to do so, but I don't understand where the logic is setting it? Or was that done already?

**Review by KaWaite — APPROVED** [review]

**Review by KaWaite — CHANGES_REQUESTED** [review]
Again...my bad, approved but I asked for english changes.

**Comment by HideBa** [comment]
@KaWaite Thanks for reviewing quickly.

File: pkg/builtin/manifest_ja.yml [hunk 3]
```diff
@@ -76,6 +76,16 @@ extensions:
             title: 色相
           surturation_shift:
             title: 彩度
+      googleAnalytics:
+        title: グーグルアナリティクス
+        description: グーグルアナリティクスを有効にし、トラッキングIDをここに貼り付けることで、公開ページがどのように閲覧されているか分析することが可能です。
```
Comment (rot1024) [inline pkg/builtin/manifest_ja.yml:81]:
```suggestion
        description: Google Analyticsを有効にすることで、公開ページがどのように閲覧されているかを分析することが可能です。
```

**Review by rot1024 — APPROVED** [review]

**Review by KaWaite — APPROVED** [review]

**Review by KaWaite — APPROVED** [review]

**Pull request merged**